<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyRepSift</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources_mainwindow.qrc">
    <normaloff>:/icons/resources/app_logo.png</normaloff>:/icons/resources/app_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main window background */
QMainWindow, QWidget {
    background-color: #a595fa;
}

/* GroupBox: slightly darker, rounded corners, SEGOE UI, 14pt, white text */
QGroupBox {

    border: 2px solid #7c6fc2;
    border-radius: 20px; /* More rounded */
    margin-top: 20px;    /* More space for the title */
    padding: 8px; /* Extra top padding for title */
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 14pt;
    color: #fff;
}

/* GroupBox title: white, bold, with background and padding */
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 20px;
    padding: 0 5px 0 5px;
    color: #fff;
    font-weight: bold;
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 14pt;
    border-radius: 10px;
}

/* Frames: transparent to let main bg show through */
QFrame {
    background: transparent;
    border: none;
}

/* Buttons: only for button1_routing and button2_conversations, white text */
QPushButton#button1_routing, QPushButton#button2_conversations {
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 14pt;
    color: #fff;
    background-color: #7c6fc2;
    border-radius: 10px;
    padding: 8px 16px;

    border: 2px solid transparent;
    transition: border 0.2s, box-shadow 0.2s;
}
QPushButton#button1_routing:hover, QPushButton#button2_conversations:hover {
    background-color: #9587e3;
    border: 2px solid #7c6fc2;
    box-shadow: 0 0 0 2px #fff;
}
QPushButton#button1_routing:pressed, QPushButton#button2_conversations:pressed {
    background-color: #6a5fb2;
}

/* Other buttons: subtle style, rounded, white text */
QPushButton {
    background-color: #7c6fc2;
    color: #fff;
    border-radius: 10px;
    padding: 6px 18px;
    font-size: 15px;

    border: 2px solid transparent;
    transition: border 0.2s, box-shadow 0.2s;
}
QPushButton:hover {
    background-color: #9587e3;
    border: 2px solid #fff;
    box-shadow: 0 0 0 2px #fff;
}
QPushButton:pressed {
    background-color: #6a5fb2;
}

/* Labels: white text for headers/footers */
QLabel#logo_header, QLabel#logo_footer {
    color: #fff;
    font-size: 18px;
    font-weight: bold;
}

/* Status bar: match main color, white text */
QStatusBar {
    background: #a595fa;
    color: #fff;
    border-top: 1px solid #7c6fc2;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources_mainwindow.qrc">:/logos/resources/header_logo.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources_mainwindow.qrc">
           <normaloff>:/icons/resources/exit.png</normaloff>:/icons/resources/exit.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignTop">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Seleccione el tipo de reporte:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
           <widget class="QPushButton" name="pushButton_2">
            <property name="statusTip">
             <string>Seleccione el archivo que empiece por &quot;HistVR&quot;</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string> Diagram Reports - CDR Routing</string>
            </property>
            <property name="icon">
             <iconset resource="../resources_mainwindow.qrc">
              <normaloff>:/icons/resources/folder_open.png</normaloff>:/icons/resources/folder_open.png</iconset>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
           <widget class="QPushButton" name="pushButton_3">
            <property name="statusTip">
             <string>Seleccione el archivo que empiece por &quot;HistChat&quot;</string>
            </property>
            <property name="text">
             <string> Interacciones y Chats - Conversaciones</string>
            </property>
            <property name="icon">
             <iconset resource="../resources_mainwindow.qrc">
              <normaloff>:/icons/resources/folder_open.png</normaloff>:/icons/resources/folder_open.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignmentFlag::AlignBottom">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources_mainwindow.qrc">:/logos/resources/footer_logo.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources_mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
